.travis.yml

```diff
@@ -1,5 +1,5 @@
+dist: xenial
```

Comment:
Could you please revert your modifications on the `.travis.yml` file ?
What you did was already merged on `lequal:dev`.

Reply:
(I will, but this will break all builds.)

Reply:
I know... but this has already been done on the dev branch.
We won't bother you with this broken build, as you're not responsible of this issue.